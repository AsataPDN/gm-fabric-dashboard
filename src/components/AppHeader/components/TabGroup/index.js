import styled from "styled-components";

const TAB_WIDTH_BASE = "1%";

const TabGroup = styled.div`
  flex: 1 1 ${TAB_WIDTH_BASE};
  display: flex;
  flex-direction: column;
  align-items: stretch;
`;
export default TabGroup;
