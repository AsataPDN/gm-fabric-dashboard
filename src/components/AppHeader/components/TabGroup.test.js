import React from "react";
import { shallow } from "enzyme";
import TabGroup from "./TabGroup";

describe("TabGroup", () => {
  it("matches snapshot", () => {
    const aTabGroup = shallow(<TabGroup />);
    expect(aTabGroup).toMatchSnapshot();
  });
});
