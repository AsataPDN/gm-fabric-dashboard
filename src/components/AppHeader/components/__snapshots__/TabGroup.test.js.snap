// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabGroup matches snapshot 1`] = `
.c0 {
  -webkit-flex: 1 1 1%;
  -ms-flex: 1 1 1%;
  flex: 1 1 1%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
}

<div
  className="c0"
/>
`;
